### ISSUE TITLE
URL bar vanishes after exiting Movie Mode (`<video controls>`) upon changing the 'Projection Mode'

### ISSUE THREAD

**Issue by cvan** [MozillaReality/FirefoxReality#761]
## Hardware
Oculus Go
54f045e

## Steps to Reproduce

1. Load https://www.youtube.com/watch?v=4dwjS_eI-lQ (or any [2D `<video>` test case](https://mzl.la/2JSKHWC))
2. Enter Fullscreen on the video player.
3. Change the Projection Mode.
4. Press the Back button on the video controls.

## Current Behavior
Notice the URL bar disappears. Nothing in the UI seems to make the URL bar appear again. Refreshing the page, re-entering Fullscreen, playing a video, etc. I could be wrong, but it appears, in my testing, the user must close the app for the URL to reappear.

![2018-11-15_01 33 01](https://user-images.githubusercontent.com/203725/48543654-d5780e00-e876-11e8-8d72-0286a3fea1bc.png)

![Animation of URL bar goes missing](https://user-images.githubusercontent.com/203725/48544385-5257b780-e878-11e8-82f3-b127dedf89c0.gif)

## Expected Behavior
The URL bar should reappear along with the other Navigation Bar UI elements when the user presses the Back button of the `Movie Mode` video controls UI.

**Comment by cvan** [comment]
@MortimerGoro can you take a look at the resizing of the widget? thanks in advance!

**Comment by cvan** [comment]
1. Launch https://webxr.sh/media
2. Open test case `#5`: [`Extra Video w/ Multiple sources, controls, and More`](https://mdn.github.io/learning-area/html/multimedia-and-embedding/video-and-audio-content/extra-video-features.html)
3. Press the video's Fullscreen icon to enter Fullscreen
4. Notice the Media Transport Controls appear
5. Press the ← back button in the Media Transport Controls
6. Notice the **URL bar disapppears**!
7. For the URL bar to reappear, **you must forcibly exit Firefox Reality** via the Oculus system's Application Manager.

**Comment by keianhzo** [comment]
@cvan can you still reproduce this? I've tried following the STRs using latest master (1120148d7fb859c2ec7990495763c81e07184621) but I couldn't reproduce it.

**Comment by cvan** [comment]
nope, but I am afraid to close this, given how severe the bug was. if I do not reproduce this by tomorrow, we can close. sound good?

**Comment by cvan** [comment]
I have given this some time, and I cannot reproduce anymore. Closing…

**Comment by cvan** [comment]
I see this again. I have reliable STR now! 👍
